modules/rest-api/pages/rest-cluster-autofailover-enable.adoc

```diff
@@ -47,6 +47,11 @@ Specifies the number of seconds that must elapse, with a node unavailable, befor
 Default setting is 120.
 The `timeout` parameter can only be specified when `enabled` is set to `true`.
 This parameter and its values are ignored if the value for the `enabled` parameter is `false`.
++
+The value of `timeout` can also be specified as 1 second.
+Note that this this low setting significantly increases the sensitivity of failure-detection
+and this, in turn, makes responses to _false failures_ more likely
+Additionally, CPU usage is likely to be considerably increased.
```

Comment:
> Additionally, CPU usage is likely to be considerably increased.

I'm not sure if we should say "considerably" as it's quite subjective. Personally I'd suggest something like
"Additionally, more CPU resource will be consumed with lower failover timeout values (below 5 seconds). Testing a representative workload and measuring CPU usage and checking for false failovers is recommended".

Reply:
I've change the explanation, hopefully to include the concepts here. Let me know what you think.